### DISCUSSION TITLE
Thumbnails in Media Browser

### DISCUSSION THREAD

**Discussion by AquaMCU** [home-assistant/frontend#11312] [Other feature requests]
Hello. 
I think it would be really nice to display thumbnails in the media browser.
When scrolling through the images in my library, I need to open each single one until I find the one that I am looking for. That is really time consuming.
It would already be a blast, if you guys could implement for images as a beginning.
Thank you for the perfect work!
Oliver

**Comment by ildar170975** [comment]
After 3 years there is no any reaction from Dev.

**Reply by ildar170975** [reply]
Also, this is a duplicate of https://github.com/home-assistant/frontend/discussions/8006 (not to mention my https://github.com/home-assistant/frontend/discussions/17955 which I closed as a duplicate).

**Comment by steverep** [comment]
Duplicate of #8006

**Comment by GeoBau** [comment]
I missing the thumbnails as well

**Comment by parautenbach** [comment]
Ditto.
